ui-integration/src/App.js

```diff
@@ -0,0 +1,298 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import "bootstrap/dist/css/bootstrap.min.css";
+
```

Comment:
```suggestion
import { saveAs } from 'file-saver';
```
for the download json - has to be installed first with `npm install file-saver`